cypress/integration/version/task_duration.ts

```diff
@@ -31,7 +31,7 @@ describe("Task Duration Tab", () => {
       cy.dataCy("leafygreen-table-row").should("have.length", 3);
       cy.location("search").should(
         "include",
-        "duration=DESC&page=0&statuses=running-umbrella,started,dispatched"
+        "duration=DESC&page=0&statuses=running-umbrella%2Cstarted%2Cdispatched"
```

Comment:
This indicates to me that the query params aren't being properly uriencoded or that we are double encoding them.

Reply:
Well isn't the URL now being encoded, and it wasn't before? query-string's [`.stringify()`](https://www.npmjs.com/package/query-string#stringifyobject-options) has encode = true as default, which is what ultimately applies the encoding.